examples/fabric/cloud_router/example_1/go/go.mod

```diff
@@ -2,11 +2,11 @@ module equinix-fabric-cloud_router-example_1
 
 go 1.21
 
-toolchain go1.22.6
+toolchain go1.21.13
 
 require (
 	github.com/equinix/pulumi-equinix/sdk latest
-	github.com/pulumi/pulumi/sdk/v3 v3.128.0
+	github.com/pulumi/pulumi/sdk/v3 v3.133.0
```

Comment:
Dang it.  The goal is to have the `upgrade-provider` action _only_ upgrade the upstream TF provider.  How can we making these changes?

Reply:
Seems like something went off the rails elsewhere; v3.133.0 was already adopted in `main` at some point, but many of the examples are using the wrong version & this is fixing those examples.

Reply:
#211 aims to get the examples back in sync.  Once that is merged we can either rebase this PR or re-run the GitHub Action to create a new one so we can see what's left to resolve.